tailcfg/tailcfg.go

```diff
@@ -818,7 +863,7 @@ func (n *Node) Equal(n2 *Node) bool {
 		n.ID == n2.ID &&
 		n.StableID == n2.StableID &&
 		n.Name == n2.Name &&
-		n.DisplayName == n2.DisplayName &&
+		n.displayName == n2.displayName &&
```

Comment:
Can a struct be compared directly like this?

Reply:
https://golang.org/ref/spec#Comparison_operators says:

> Struct values are comparable if all their fields are comparable. Two struct values are equal if their corresponding non-blank fields are equal.